gabbi/httpclient.py

```diff
@@ -172,15 +172,10 @@ def _verbose_output(self, message, prefix='', color=None, stream=None):
 
 def get_http(verbose=False, caption=''):
     """Return an Http class for making requests."""
-    if verbose:
-        body = True
-        headers = True
-        colorize = True
-        stream = sys.stdout
-        if verbose == 'body':
-            headers = False
-        if verbose == 'headers':
-            body = False
-        return VerboseHttp(body=body, headers=headers, colorize=colorize,
-                           stream=stream, caption=caption, strict=True)
-    return Http(strict=True)
+    if not verbose:
+        return Http(strict=True)
+
+    headers = False if verbose == 'body' else True
+    body = False if verbose == 'headers' else True
```

Comment:
FWIW, this seems like a good (rare) example of ternaries actually being significantly more readable than the less concise alternative ("include headers/body unless verbosity is set to body/headers").

Reply:
🤷‍♂️

Reply:
You might not care, but I'll be sure to weaponize this in future discussions.